docs/octant-plugin-installation.md

```diff
@@ -113,3 +117,17 @@ You can follow the steps listed below to install octant and antrea-octant-plugin
     ```
 
 Now, you are supposed to see Octant is running together with antrea-octant-plugin via URL http://(IP or $HOSTNAME):80.
+
+Note:
+1. For Traceflow UI, it is a separate octant plugin called antrea-traceflow-plugin in Antrea release v0.8.0, but
+we plan to merge it into antrea-octant-plugin soon. You can already view the alpha version of traceflow UI if you chooose
+to deploy Antrea UI as a Pod, since we have already included this plugin into image antrea/octant-antrea-ubuntu.
+2. If you deploy Antrea UI as a process, you can not see Traceflow UI for now if you follow the five steps listed above,
+since we have not merged this function into antrea-octant-plugin in Antrea v0.8.0 release assets yet. But you can still
+compile the binary by yourself via command below for step 3 and the remaining steps are almost the same as antrea-octant-plugin.
```

Comment:
suggestion:
1. In Antrea v0.8.0, the Traceflow UI is a separate Octant plugin called `antrea-traceflow-plugin`. Starting with v0.9.0, the Traceflow UI will be merged into `antrea-octant-plugin`. When deploying Octant as a Pod using image `antrea/octant-antrea-ubuntu:v0.8.0`, you already have access to the alpha version of the Traceflow UI.
2. If you deploy Octant and the Antrea UI as a process, you cannot access the Traceflow UI for now when following the steps listed above (at least until the v0.9.0 release). However, you can still build the binary yourself with the command below, with the remaining steps being almost the same as the ones above.

We just need to remember to update this after https://github.com/vmware-tanzu/antrea/pull/913 is merged

Reply:
Thanks. Have updated as suggested, please help review the latest version.